src/bundle/Resources/config/ezplatform_default_settings.yml

```diff
@@ -27,3 +27,6 @@ parameters:
 
     # User identifier
     ezsettings.default.user_content_type_identifier: ['user']
+
+    # Copy subtree children limit
+    ezsettings.default.application_config.copy_subtree.limit: 100
```

Comment:
@ViniTou this is not an ordinary parameter. It's meant to be dynamic SiteAccess-aware setting. It needs prepended semantic configuration, otherwise there's no way to configure it. 

Moreover in this case, `application_config` node is something new, breaking convention, so I'm not entirely convinced about it. cc @webhdx.